qutebrowser/browser/webelem.py

```diff
@@ -42,7 +42,7 @@
 SELECTORS = {
     Group.all: ('a, area, textarea, select, input:not([type=hidden]), button, '
                 'frame, iframe, link, [onclick], [onmousedown], [role=link], '
-                '[role=option], [role=button], img'),
+                '[role=option], [role=button], img, [ng-click]'),
```

Comment:
The [stackoverflow post](https://stackoverflow.com/questions/41834600/adding-angular-2-click-attribute-globally-to-css) you linked mentions `[ngClick], [data-ng-click], [x-ng-click]` for Angular 1, which is not what you use here - why is that?

Reply:
I tottaly forgot about those. Will add them now. 

Do you also want me to add tests for those?